vizro-core/examples/visual-vocabulary/requirements.in

```diff
@@ -3,5 +3,5 @@ autoflake==2.3.1
 black==24.4.2
 isort==5.13.2
 plotly==5.24.1
-vizro==0.1.25
+vizro==0.1.27
```

Comment:
Need to update to vizro 0.1.28 when available

Reply:
Maybe obvious, but don't forget to generate the requirements.txt as well.